ni/measurementlink/sessionmanagement/v1/session_management_service.proto

```diff
@@ -118,6 +118,34 @@ message SessionInformation{
   // This field is empty for any SessionInformation returned from ReserveAllRegisteredSessions.
   // This field is readonly.
   repeated ChannelMapping channel_mappings = 6;
+  
+  // Optional. Represents a set of annotations related to client machine details for session reservation or registration.
```

Comment:
I'm inclined to just document this as: 

"Optional. A set of annotations attached to the session. For example, the annotations can provide a description of the session or identify who reserved a session."

and not document well known annotations at this time. I think we should view any annotations as implementation details at this point and not as a formal API that is guaranteed to be there or will never change. The same goes for multiplexer sessions.